diff --git a/node_modules/prebid.js/src/refererDetection.js b/node_modules/prebid.js/src/refererDetection.js
index 6019867..e27b90b 100644
--- a/node_modules/prebid.js/src/refererDetection.js
+++ b/node_modules/prebid.js/src/refererDetection.js
@@ -8,6 +8,12 @@
  * Canonical URL which refers to an HTML link element, with the attribute of rel="canonical", found in the <head> element of your webpage
  */
 
+// @tab-ads-patch
+// We modify this to use the referer info from the Prebid config
+// because our page may be loaded in a new tab page iframe, which
+// can break referer collection.
+// TODO
+
 import { logWarn } from './utils.js';
 
 export function detectReferer(win) {
